clang/test/Driver/freebsd.c

```diff
@@ -203,3 +203,12 @@
 // RELOCATABLE-NOT: "-l
 // RELOCATABLE-NOT: crt{{[^./\\]+}}.o
 
+// Check that the -X flag is passed to the linker on riscv64
```

Comment:
The CHECK is self-explanatory and renders the comment redundant. The RUN line may test additional options in the future.

Reply:
When I was putting the diff together I was questioning whether I should combine the two into one instance.